lms/djangoapps/certificates/models.py

```diff
@@ -745,10 +761,20 @@ class CertificateTemplateAsset(TimeStampedModel):
     )
     asset = models.FileField(
         max_length=255,
-        upload_to='certificate_template_assets',
+        upload_to=template_assets_path,
         help_text=_(u'Asset file. It could be an image or css file.'),
     )
 
+    def save(self, *args, **kwargs):
```

Comment:
@ziafazal I am saving the file in method `template_assets_path` at the following location  -->  `certificate_template_assets/{model_identifier}/filename`. 
*model_identfier* is none when we create a new record for `CertificateTemplateAsset` . So that's why i override the `save` method. model_identifier should not be none.